### ISSUE TITLE
Page.get_texttrace return fontsize 0

### ISSUE THREAD

**Issue by little-polka-dot** [pymupdf/PyMuPDF#2506]
## Describe the bug (mandatory)
Method "Page.get_texttrace" return fontsize 0 when the character is rotated 90 or -90 degrees in this Pdf.

## To Reproduce (mandatory)
[output.pdf](https://github.com/pymupdf/PyMuPDF/files/11904992/output.pdf)

```python
d = fitz.Document('output.pdf')
for i in d[0].get_texttrace():
    # print(i)
    print('chars:', [chr(i[0]) for i in i['chars']], 'fontsize:', i['size'])

origin = d[0].get_textpage().extractDICT()
for bi in origin['blocks']:
    for li in bi['lines']:
        for si in li['spans']:
            print('chars:', si['text'], 'size:', si['size'])
``` 

## Expected behavior (optional)
Describe what you expected to happen (if not obvious).

## Screenshots (optional)
Pdf:
![image](https://github.com/pymupdf/PyMuPDF/assets/44431347/75c9993f-c013-4519-aae4-a7170c08e59b)
Output:
![image](https://github.com/pymupdf/PyMuPDF/assets/44431347/e82d1d02-ca54-4f9a-aab0-b5e5aed7ba18)

## Your configuration (mandatory)
3.7.9 (tags/v3.7.9:13c94747c7, Aug 17 2020, 18:58:18) [MSC v.1900 64 bit (AMD64)] 
 win32 
 
PyMuPDF 1.22.5: Python bindings for the MuPDF 1.22.2 library.
Version date: 2023-06-21 00:00:01.
Built for Python 3.7 on win32 (64-bit).

**Comment by little-polka-dot** [comment]
Whats relationship between block-number returned by extractRAWDICT() and seqno returned by get_texttrace()

**Comment by JorjMcKie** [comment]
> Whats relationship between block-number returned by extractRAWDICT() and seqno returned by get_texttrace()

There is no relationship - at all.
`get_text()` is based on a `TextPage` object, which is created by MuPDF using complex heuristics to come up with the hierarchy `block -> line -> span -> char`. Plus, as optional services, generation of spaces, conversion of white, detection of hyphenation.

**Comment by JorjMcKie** [comment]
Found and fixed the error. Will be rolled out in a near-future version.

**Comment by little-polka-dot** [comment]
Thank you. And i have another question, whats difference between seqno returned by get_texttrace() and seqno returned by get_drawings?😊

**Comment by JorjMcKie** [comment]
> Thank you. I have another question, whats difference between seqno returned by get_texttrace() and seqno returned by get_drawings?

The sequence numbers in both cases are the running number of the executed command. And they are consistent across the methods, such that a smaller seqno in one means the corresponding command is executed earlier than another one in either list with a larger seqno.
The seqno also equals the list index of the corresponding bbox in `page.get_bboxlog()`.
So this should help to tell if something is "covered" by an object if the bboxes overlap.

**Comment by JorjMcKie** [comment]
Fixed prior to the current version 1.23.4.
